pykokkos/__init__.py

```diff
@@ -45,8 +45,23 @@
                                  fmin,
                                  exp,
                                  exp2,
-                                isinf,
-                                isnan)
+                                 argmax,
+                                 unique,
+                                 var,
+                                 in1d,
+                                 mean,
+                                 sum,
```

Comment:
Just a drive-by comment for now, but I'm pretty sure that `sum()` is not a `ufunc`--it shouldn't be listed in the NumPy docs under available ufuncs: https://numpy.org/doc/stable/reference/ufuncs.html#available-ufuncs

nor should it be listed as a ufunc in our checklist at gh-62. I think there's a distinction into a separate category for the pure reduction operations, so we may want to reflect that in our library/module organization.